viral/response_profiles.py

```
@@ -0,0 +1,179 @@
+            do_shuffle=False,
+            threshold_speed=False,
+            bin_occupancy_divide=True,
+        )
+        signal.append(data)
+    return np.array(signal)
+
+
+def batch_process() -> None:
+
+    for mouse_name, dates in tqdm(SESSIONS_KEEP.items()):
+        for stage, date in dates.items():
```

Ruff (0.14.14)

[warning] Loop control variable `stage` not used within loop body

Rename unused `stage` to `_stage`

(B007)